apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/StartSpanOptions.cs

```diff
@@ -22,16 +25,22 @@ public sealed class StartSpanOptions
         /// <summary>Gets the span kind.</summary>
         public SpanKind SpanKind { get; }
 
-        private StartSpanOptions(SpanKind spanKind)
+        /// <summary>Gets the lables to be added to the span.</summary>
```

Comment:
lables => labels

Perhaps this change should be in a separate PR? It's not obvious to me how it relates to the IDisposable aspect.